webapp/components/Editor/Editor.vue

```diff
@@ -279,6 +300,82 @@ export default {
               return fuse.search(query)
             },
           }),
+          new Hashtag({
+            // a list of all suggested items
+            items: () => {
+              return this.hashtags
+            },
+            // is called when a suggestion starts
+            onEnter: ({ items, query, range, command, virtualNode }) => {
```

Comment:
Here seems to be alot of duplicate code - is it possible to make an mention/tag-agnostic component?

Reply:
I have an own component  in mind for the future. @ulfgebhardt 
With @appinteractive help I tried to make a class out of mention/hasgtag, but I could not get it to work, because the functions here are reactive and deep connected to the editor component. My skills in JavaScript are not good enough, may be.

I played around with this to long now and like to try that later.

Reply:
Ok - awesome!